ledger/eval.go

```diff
@@ -1032,6 +1032,41 @@ func (eval *BlockEvaluator) endOfBlock() error {
 			eval.block.TxnCounter = 0
 		}
 
+		// We are going to find the list of modified accounts and the
+		// current round that is being evaluated.
+		// Then we are going to go through each modified account and
+		// see if it meets the criteria for adding it to the expired
+		// participation accounts list.
+		modifiedAccounts := eval.state.mods.Accts.ModifiedAccounts()
+		currentRound := eval.Round()
+
+		proto, ok := config.Consensus[eval.block.BlockHeader.CurrentProtocol]
+		if !ok {
+			return fmt.Errorf("unknown consensus version: %s", eval.block.BlockHeader.CurrentProtocol)
+		}
+
```

Comment:
Use `eval.proto` instead of getting the protocol parameters from `config.Consensus`